README.rst

```diff
@@ -0,0 +1,107 @@
+courseraoauth2client
+====================
+
+.. image:: https://travis-ci.org/coursera/courseraoauth2client.svg
+    :target: https://travis-ci.org/coursera/courseraoauth2client
+
+This project is a library consisting of a command line interface and a client
+for interacting with Coursera's OAuth2 authorized APIs.
+
+Installation
+------------
+
+To install this sdk, simply execute::
+
+    sudo pip install courseraoauth2client
+
+`pip <https://pip.pypa.io/en/latest/index.html>`_ is a python package manager.
+If you do not have ``pip`` installed on your machine, please follow the
+installation instructions for your platform found at:
+https://pip.pypa.io/en/latest/installing.html#install-or-upgrade-pip
+
+Setup
+-----
+
+Before using Coursera's OAuth2 APIs, be sure you know your client id,
+client secret, and scopes you want for for your application. You may create
+an application at https://accounts.coursera.org/console. When creating the
+application, set the
+``Redirect URI`` to be ``http://localhost:9876``.
+
+Then put your client id, client secret, and scopes
+in a file at ``~/.coursera/courseraoauth2client.cfg``::
```

Comment:
So, the problem with this, is that if you have 2 apps that use this library, they both would end up stomping on each other for this same file. Could you look for maybe a better way to do this? (i.e. load up a file that's included in the package of the scripts that use it?) You'd probably need a fallback for interactive execution tho...